assets/javascripts/src/modules/analytics/setup.js

```diff
@@ -31,6 +31,9 @@ define([
     function setupAnalytics() {
         ophan.init();
         ga.init();
+        ophan.ophan.then(function(ophan){
+            ga.setOphanId(ophan.pageViewId);
```

Comment:
Can't you just fire ga.init() here? If you only call 'setOphanId' here then it gets set after the pageview and won't be sent to Google. You'd have to fire pageview twice to make it send, which isn't appropriate.